### PR TITLE
GIX-2118: Add ckETH logos to universe

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Use the new ETH logos for ckETH related universes.

# Changes

* Import and use the ckETH logos in the icrc-universes derived store.

# Tests

* Change the mocks to use the new logos. Those are used in test expects.

# Todos

- [ ] Add entry to changelog (if necessary).

Not necessary.

### CODE REVIEW

**Comment by lmuntaner** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3874** <a href="https://app.graphite.dev/github/pr/dfinity/nns-dapp/3874?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/dfinity/nns-dapp/3874?utm_source=stack-comment).

**Comment by lmuntaner** [comment]
@dskloetd @peterpeterparker please review

File: frontend/src/lib/derived/icrc-universes.derived.ts [hunk 1]
```diff
@@ -23,12 +23,11 @@ const convertIcrcCanistersToUniverse = ({
...
-  // TODO: Add ckETH and ckETHTEST logos
   const logo =
     universeId === CKETH_UNIVERSE_CANISTER_ID.toText()
-      ? CKBTC_LOGO
+      ? CKETH_LOGO
       : universeId === CKETHSEPOLIA_UNIVERSE_CANISTER_ID.toText()
-      ? CKTESTBTC_LOGO
+      ? CKSEPOLIAETH_LOGO
       : undefined;
   if (isNullish(token) || isNullish(logo)) {
```
Comment (peterpeterparker) [inline frontend/src/lib/derived/icrc-universes.derived.ts:32]: is it correct to exclude an universe correctly configured but for which only the logo would not be provided? UI we can fallback on a placeholder, we do that in Oisy Wallet or VRC.

**Review by peterpeterparker — APPROVED** [review]
PR LGTM. A question regarding existing code.

File: frontend/src/lib/derived/icrc-universes.derived.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/lib/derived/icrc-universes.derived.ts:32]: Yes, that sounds good once we have a default logo.

But ideally we'll get the logo from the token metadata. But I guess we will still need the default logo.

**Pull request merged**